static/app/bootstrap/index.tsx

```diff
@@ -75,7 +76,7 @@ async function promiseRequest(url: string) {
 }
 
 function preloadOrganizationData(config: Config) {
-  if (!config.shouldPreloadData) {
+  if (!config.shouldPreloadData || EXPERIMENTAL_SPA) {
```

Comment:
this function only runs for EXPERIMENTAL_SPA so in essence you've just turned this feature off

Reply:
you're right, @scttcper! That’s why I was running into errors after accepting the invite and landing on the org page. I’ve updated the code to use regex to check for invite routes using pure javascript + window. I’ve just tested everything, and it’s all working smoothly now.